src/allmydata/test/test_version.py

```diff
@@ -272,4 +272,4 @@ def test_report_import_error(self):
 
         foolscap_stuffs = [stuff for (pkg, stuff) in vers_and_locs if pkg == 'foolscap']
         self.failUnlessEqual(len(foolscap_stuffs), 1)
-        self.failUnless([e for e in errors if "dependency \'foolscap\' could not be imported" in e])
```

Comment:
Under Python 2 the message is now: `dependency u'foolscap' could not be imported.`